app/server/lib/DocApi.ts

```diff
@@ -595,6 +603,27 @@ export class DocWorkerApi {
       archive.dataStream.pipe(res);
     }));
 
+    this._app.post('/api/docs/:docId/attachments/archive', isOwner, withDoc(async (activeDoc, req, res) => {
```

Comment:
This is tricky to test. Ideally, I think I want to:
- Download the document with external attachments
- Reupload it
- Run the attachment upload

But I can't see an endpoint for uploading documents in our docs. Do we have one?